packages/python/accern_xyme/accern_xyme.py

```diff
@@ -2138,6 +2144,16 @@ def get_path(self, *path: str) -> 'BlobHandle':
         return BlobHandle(
             self._client, f"{self._uri}/{'/'.join(path)}", is_full=True)
 
+    def get_parent(self) -> 'BlobHandle':
+        if self._parent is None:
```

Comment:
![image](https://user-images.githubusercontent.com/7776115/119029730-015c3900-b977-11eb-994f-1209913f3324.png)
is this what you are trying to do?

Reply:
also add get_parent to todo_api for typescript

Reply:
yes